Assets/Scripts/Game/InputManager.cs

```diff
@@ -79,6 +79,7 @@ public class InputManager : MonoBehaviour
         KeyCode[] heldKeys = new KeyCode[totalHeldKeys];
         KeyCode[] previousKeys = new KeyCode[totalHeldKeys];
         KeyCode heldModifier;
+        KeyCode swingWeaponBinding = KeyCode.Mouse1;
```

Comment:
All changes in InputManager can be removed from the if-statement comment suggestion.